### PR TITLE
CImg: patch inpaint.h

### PR DESCRIPTION
**Pull request by devernay**
see https://github.com/NatronGitHub/openfx-misc/blob/f03c6c0d75dcc01034355c46760b98a8ff76691f/CImg/Makefile#L190

### CODE REVIEW

**Comment by nedrichards** [comment]
@devernay as you and @Sunderland93 have push rights on this repo either of you can type bot, build in PRs here now to run test builds that should report back here - but sometimes you have to go and check https://flathub.org/builds/ (unreliable bot!) then merge when everything's green.

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
https://flathub.org/builds/#/builders/2/builds/1844

File: fr.natron.Natron.json [hunk 1]
```diff
@@ -844,6 +844,7 @@
                     "commands": [
                         "cp ./CImg-v.2.2.3/CImg.h ./CImg",
                         "cp ./CImg-v.2.2.3/plugins/inpaint.h ./CImg/Inpaint"
```
Comment (Sunderland93) [inline fr.natron.Natron.json]: Missing ","

**Comment by devernay** [comment]
bot, build

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
built fine: https://flathub.org/builds/#/builders/2/builds/1859

**Pull request merged**